sources/php5/KalturaClientBase.php

```diff
@@ -706,7 +706,7 @@ public function setClientConfiguration(KalturaClientConfiguration $configuration
 		$params = get_class_vars('KalturaClientConfiguration');
 		foreach($params as $param)
 		{
-			if(is_null($configuration->$param))
+			if(empty($configuration->$param) || is_null($configuration->$param))
```

Comment:
disregard last comment.. I read it incorrectly.
It checks whether *$this->clientConfiguration[$param]* is set, NOT *$configuration->$param*